core-foundation/src/string.rs

```diff
@@ -136,10 +137,8 @@ impl CFString {
 
     /// Returns the number of characters in the string.
     #[inline]
-    pub fn char_len(&self) -> CFIndex {
-        unsafe {
-            CFStringGetLength(self.0)
-        }
+    pub unsafe fn char_len(&self) -> CFIndex {
+        CFStringGetLength(self.0)
```

Comment:
Yes I understand that, but why did you remove the unsafe block inside `char_len` and instead made `char_len` unsafe?